### DISCUSSION TITLE
A little love for the subscription section of our dashboard

### DISCUSSION THREAD

**Discussion by Kriz02** [gameserverapp/Platform#946] [Suggestions]
Would be nice to have a few new features to this.

- The drop down box for selecting which character a subscription belongs too, make it like the new drop down for the shop system where it shows the player, and map they were last on, or at the very least what game the character belongs too. Get a lot of players select a character for the wrong game and makes the delivery fail. Because of this we have to do trial and error if all characters are the same name until we find the right one, this can only be changed then we have to wait for the next monthly cycle to see if it is the correct one. 

- Perhaps add the players e-mail or a better way for us to find selected players (even adding their subscription character name to the customer id on PayPal for the subscription would help if this is possible, this is what I try to do manually as new ones come in but lately we are having so many new subscriptions I just don't have time to keep up with it)

- Make it so we can search subscript id, player names, subscriptions, etc. Just something to make it easier to find players in the sea of subscriptions. For us personally we have 44 pages of subscribers (new, old, expired) and when we need to look one up by PayPal ID, it really gets cumbersome to search page by page.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestions. Could you please post each suggestion in a separate item?

Will reopen when you did
